### PR TITLE
incidental typo fix

### PR DESCRIPTION
**Pull request by Mrt134**
Thanks for contributing to TrueNAS documentation! By opening a Pull Request, you're acknowledging that your changes will be distributed under the [Creative Commons 4.0](https://creativecommons.org/licenses/by-nc-sa/4.0/) license.

### CODE REVIEW

File: content/SCALE/SCALETutorials/Shares/SMB/AddSMBShares.md [hunk 1]
```diff
@@ -269,4 +269,4 @@ Mount the volume. `sudo mount_smbfs -I computer_name\share_name /mnt/smb_share`.
 {{< /expand >}}
 
 {{< taglist tag="scalesmb" limit="10" >}}
-{{< taglist tag="scaleafp" limit="10" title="Releated AFP Articles" >}}
\ No newline at end of file
+{{< taglist tag="scaleafp" limit="10" title="Releated AFP Articles" >}}
```
Comment (clairewrtr) [inline content/SCALE/SCALETutorials/Shares/SMB/AddSMBShares.md:272]:
```suggestion
{{< taglist tag="scaleafp" limit="10" title="Related AFP Articles" >}}
```

File: content/SCALE/SCALETutorials/Shares/SMB/AddSMBShares.md [hunk 2]
```diff
@@ -203,7 +203,7 @@ The **Hosts Allow** and **Hosts Deny** fields work together to produce different
 {{< /expand>}}
 
 #### Approving Apple Software Compatibility
```
Comment (clairewrtr) [inline content/SCALE/SCALETutorials/Shares/SMB/AddSMBShares.md:205]:
```suggestion
#### Improving Apple Software Compatibility
```

**Review by clairewrtr — CHANGES_REQUESTED** [review]
Just a few questions...

**Comment by Mrt134** [comment]
Updated, thanks!

**Review by clairewrtr — APPROVED** [review]
Looks good.

**Pull request merged**